wiki/fr/fr-Onboarding.md

```diff
@@ -1,57 +0,0 @@
----
```

Comment:
How much did we edit on this file? I think it's an important one. @jujudusud @trebmuh @gegeweb should we remove this one?

Reply:
I will update this one evening if everybody's ok to align with en file state.

Reply:
@jujudusud Please can you confirm that you are happy to keep updating it when substantial changes are introduced fairly soon? The point of this ticket is that we want to avoid putting undue pressure on translator while the English version is still settling down.

Reply:
Hi @gilgongo, I confirm that I am happy to keep updating it when substantial changes are introduced fairly soon.
Will do my best for the best result. ;-)